### ISSUE TITLE
Accessibility and Text change on collection access settings form

### ISSUE THREAD

**Issue by astridu** [sul-dlss/hungry-hungry-hippo#606]
1. The grouped form controls (groups of radio buttons) are missing a name. 
2. It says the last 2 radio buttons for the second question do not have a label, but it looks ok to me.
3. Please change the text at the top of the page to an H2: "Manage release of deposits for discovery and download"
4. The 3 questions "When will files...", "Who will have permission...", "Will a DOI be assigned.." should be bold Paragraph.

**Comment by justinlittman** [comment]
Note: Title already changed to H2.
